data/brands/amenity/atm.json

```diff
@@ -240,6 +240,16 @@
         "name:he": "בנק טינקוף",
         "name:ru": "Тинькофф Банк"
       }
-    }
+    },
+    {
```

Comment:
https://github.com/osmlab/name-suggestion-index/blob/0f5370d5d28a099e595a2cb66d67227b567ab308/data/brands/amenity/atm.json#L2-L15

ATM will inherit from bank.json, you don't need to add twice, just add in bank.json will affect both amenity=bank and amenity=atm.

Reply:
Oh, ok